action/scenario_actions/include/scenario_actions/action_manager.h

```diff
@@ -4,13 +4,17 @@
 #include <memory>
 #include <vector>
 
-#include <pluginlib/class_loader.h>
+#include <pluginlib/class_loader.hpp>
 
 #include <yaml-cpp/yaml.h>
 
 #include <scenario_actions/entity_action_base.h>
-#include <scenario_api/scenario_api_core.h>
-#include <scenario_intersection/intersection_manager.h>
+
+// Forward declarations
+class ScenarioAPI;
+namespace scenario_intersection {
+class IntersectionManager;
+}
```

Comment:
Could we track these forward declarations somewhere so that they can be replaced with the actual headers once their ports are complete. For this purpose, commenting out the headers rather than deleting might be more convenient for the replacement.

Reply:
Is there any drawback to forward declarations?

Reply:
In theory no, but it's a smell that the dependencies in the code are perhaps too complex.

Reply:
I removed the forward declarations.